Refactored admin into frontend

navindu-sachintha opened the pull request:
- **Added config api endpoints**
- **Added src**
- **Migrated ragapp pages router to App router**
- **Added RAG app admin panel**
- **backend changes for admin panel**
- **admin changes for admin panel**
- **refined UI**
- **deleted key**
- **♻️ Admin Panel Refactor**

navindu-sachintha committed:
Added config api endpoints

navindu-sachintha committed:
Added src

navindu-sachintha committed:
Migrated ragapp pages router to App router

navindu-sachintha committed:
Added RAG app admin panel

NilaanL committed:
backend changes for admin panel

NilaanL committed:
admin changes for admin panel

nilaan-effectz committed:
Merge pull request #35 from effectz-ai/adminpanel_backend_NL

Adminpanel backend nl

navindu-sachintha committed:
refined UI

navindu-sachintha committed:
deleted key

navindu-sachintha committed:
♻️ Admin Panel Refactor

navindu-sachintha committed:
♻️ Updated refactor

navindu-sachintha committed:
➖  deleted secret

navindu-sachintha committed:
Merge branch 'main' into admminReafactor_NS

# Conflicts:
#	.gitignore
#	effectzgpt/admin/package-lock.json
#	effectzgpt/backend/.env
#	effectzgpt/backend/README.md
#	effectzgpt/backend/app/engine/__init__.py
#	effectzgpt/backend/app/engine/generate.py
#	effectzgpt/backend/main.py
#	effectzgpt/backend/poetry.lock
#	effectzgpt/backend/pyproject.toml

navindu-sachintha committed:
Merge branch 'main' into admminReafactor_NS

# Conflicts:
#	effectzgpt/frontend/package-lock.json
#	effectzgpt/frontend/package.json

NilaanL committed:
Clean refactoring changes

NilaanL committed:
Merge branch 'dev' into admminReafactor_NS

# Conflicts:
#	effectzgpt/backend/config/loaders.yaml
#	effectzgpt/backend/poetry.lock
#	effectzgpt/backend/pyproject.toml

nilaan-effectz commented:
@navindu-sachintha  verify the changes in the branch with the main , and test main funtions

NilaanL committed:
Merge branch 'dev' into admminReafactor_NS

NilaanL committed:
refactored admin panel , added folder structure for knowledge base
introduced new env variable NEXT_PUBLIC_CHAT_API_BASE_URL

NilaanL committed:
introduced new env variable NEXT_PUBLIC_CHAT_API_BASE_URL removed NEXT_PUBLIC_CHAT_API

NilaanL committed:
update dependencies

NilaanL committed:
update depreciated method in BaseModel

NilaanL committed:
parse TOP_K env variable to int and add missing node_postprocessors variable

NilaanL committed:
update depreciated method copy() in BaseEnvConfig

NilaanL committed:
refactoring , remove unused imports

NilaanL committed:
Fix load_configs method usage

NilaanL committed:
FileHandler | refactor get_current_files method ;
add new method list_nested_files_and_folders in FileHandler ;
add fileIsExist check to upload_file method ;
refactored remove_file method

File : added two properties type , parent

NilaanL committed:
update file Delete endpoint
